### PR TITLE
Add platform specific freertos fetch script

### PR DESCRIPTION
**Pull request by hihigupt**

### CODE REVIEW

**Comment by hihigupt** [comment]
https://github.com/Azure-Samples/iot-middleware-freertos-samples/issues/40

File: .github/scripts/fetch_freertos.ps1 [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+param (
+    [ Parameter( Position=0 ) ]
+    [ string ] $TEST_FREERTOS_SRC = "libs/FreeRTOS"
```
Comment (danewalton) [inline .github/scripts/fetch_freertos.ps1:6]: nit: should this be `TEST_`? Should it be `FREERTOS_SRC_DIR`?

File: .github/scripts/fetch_freertos.ps1 [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+)
+
+Write-Output "::group::FreeRTOS source"
+$TEST_FREERTOS_COMMIT_ID="c8fa483b68c6c1149c2a7a8bc1e901b38860ec9b"
```
Comment (danewalton) [inline .github/scripts/fetch_freertos.ps1:10]: Same here. Just `FREERTOS_COMMIT_ID`?

File: .github/scripts/fetch_freertos.ps1 [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+if ( !( Test-Path -Path $TEST_FREERTOS_SRC ) )
+{
+    git clone https://github.com/FreeRTOS/FreeRTOS.git $TEST_FREERTOS_SRC
+    cd libs/FreeRTOS
```
Comment (danewalton) [inline .github/scripts/fetch_freertos.ps1:15]:
```suggestion
    cd $TEST_FREERTOS_SRC
```

**Review by danewalton — APPROVED** [review]

**Review by ewertons — APPROVED** [review]

**Pull request merged**